package.json

```
@@ -13,9 +13,9 @@
     "destroy": "rm -rf node_modules package-lock.json dist",
     "staging:release": "npm run build && npm run deploy:version:s3 && npm run deploy:next:s3 && npm run deploy:next:npm",
     "release": "npm run build && npm run deploy:version:s3 && npm run deploy:latest:s3 && npm run deploy:latest:npm",
-    "deploy:next:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.min.js --acl public-read",
+    "deploy:next:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.min.js --acl public-read && aws cloudfront create-invalidation --distribution-id E2FG4SLRW595GE --paths /asset-manager/releases/next/*",
```

Sorry again. Can we make distribution ID a secret?
```suggestion
    "deploy:next:s3": "aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/next/webloader.min.js --acl public-read && aws cloudfront create-invalidation --distribution-id $DISTRIBUTION_ID --paths /asset-manager/releases/next/*",
```